lib/services/authority.go

```diff
@@ -287,6 +287,9 @@ type UserCertParams struct {
 	// MFAVerified is the UUID of an MFA device when this Identity was
 	// confirmed immediately after an MFA check.
 	MFAVerified string
+	// MFAVerified is the the hard deadline of a session when this Identity was
```

Comment:
Same comment as above. It's not a hard deadline, because if "disconnect expired cert" is disabled then it does absolutely nothing.

How about something like `OriginalIdentityExpires`?

Reply:
https://github.com/gravitational/teleport/pull/18607#discussion_r1028409180